graviti/openapi/requests.py

```diff
@@ -55,5 +55,5 @@ def open_api_do(method: str, access_key: str, url: str, **kwargs: Any) -> Respon
         return do(method=method, url=url, **kwargs)
     except ResponseError as error:
         response = error.response
-        error_code = response.json()["code"]
+        error_code = response.json().get("code", "InternalServerError")
```

Comment:
shall we use a new Error code for status code `503`
Also, shall we also handle `404` error here?